src/amo/pages/Addon/index.js

```diff
@@ -109,13 +101,15 @@ export class AddonBase extends React.Component {
     // This makes sure we do not try to dispatch any new actions in the case
     // of an error.
     if (!errorHandler.hasError()) {
-      const { slug } = params;
```

Comment:
Why did you remove this? It just means you have to use `params.slug` twice below, so it seems to have introduced churn without any real benefit.

Reply:
I find it more readable because `params.slug` might not be a `slug`. Having `params.` makes it easier to me to understand that it's the name of the parameter in the URL.